Dockerfile

```diff
@@ -3,20 +3,47 @@
 # in order to appropriately inject the supervisord binary into the application container.
 #
 
+# SUPERVISORD
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS supervisordbuilder
```

Comment:
I guess that this is supposed to be a "upstream" Dockerfile.
This image can't be downloaded without first logging to ci.opesnhfit.org